util/__init__.py

```diff
@@ -1140,6 +1140,7 @@ def parse(cls, amount):
         currency = cls.DEFAULT_CURRENCY
         if not amount:
             amount = '0'
+        amount = unicode(amount)
```

Comment:
Not sure about this one, but I recall circulation code was sometimes passing a string into this method and sometimes a number.